opengcc/joybus.cpp

```diff
@@ -156,6 +156,14 @@ void send_mode(uint8_t mode) {
     sticks sticks_copy = state.analog_sticks;
     triggers triggers_copy = state.analog_triggers;
 
+    if (mode != 0x06 && !state.center_set) {
```

Comment:
Consider implications of this on Wii, determine if acceptable

Reply:
Also, look into trigger centering, right now it means on console the triggers always read ~30 minimum

Reply:
Also, look into trigger configuration bug where held trigger displays it's digital value.